index.html

```diff
@@ -1,14 +1,9 @@
 <!DOCTYPE html>
 <html>
 <head>
-<!-- Compiled and minified CSS -->
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.2/css/materialize.min.css">
-  <link rel="stylesheet" href="main.css">
-
-
-  <!-- Compiled and minified JavaScript -->
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.2/js/materialize.min.js"></script>
-
+    <!-- Compiled and minified CSS -->
```

Comment:
I can't tell if you're using tabs in this file; if you are please convert to spaces.